src/main/java/com/vadmack/mongodbtest/service/ProjectService.java

```diff
@@ -91,4 +103,10 @@ private Project getById(Long id) {
         return repository.findById(id)
                 .orElseThrow(() -> new NotFoundException(String.format("Project with id=%d not found", id)));
     }
+
+    private Long getUserId() {
```

Comment:
You can get the authentication principle using `@AuthenticationPrincipal` annotation. Let's try to use